app/services/search/query_builders/feed_content.rb

```diff
@@ -0,0 +1,78 @@
+module Search
+  module QueryBuilders
+    class FeedContent < QueryBase
+      QUERY_KEYS = %i[
+        search_fields
+      ].freeze
+
+      TERM_KEYS = {
+        tag_names: "tags.name",
```

Comment:
Within our Feed Content docs we have nested objects such as tags and users. This means to search those object's attributes we have to present Elasticsearch with the "path" so it can find the attribute we want. This hash allows us to map a single key from our controller with the correct search term. 

While this breaks from what we have done with the other models I do really like this approach for a couple of reasons. 
* It conceals the actual fields we have in Elasticsearch from the outside
* It allows us to change the structure of Elasticsearch documents or what we search without having to change the frontend code. I ended up doing this MANY times at my old job and only having to change a value in a single hash rather than all the front end code was a HUGE timesaver.

Reply:
+1 on putting this explanation in a comment embedded in the code

Reply:
I am going to have to map the term keys as well. I will throw a comment in with my next PR when I make that change!